ResearchKit/Common/ORKAnswerFormat.m

```diff
@@ -908,6 +929,10 @@ - (Class)questionResultClass {
     return [ORKBooleanQuestionResult class];
 }
 
+- (NSString*)stringForAnswer:(id)answer {
+    return [self.impliedAnswerFormat stringForAnswer: @[answer]];
+}
```

Comment:
Is this override necessary? Since the implied answer format is already implemented in this class, and the super class's (ORKAnswerFormat) implementation redirects the `stringForAnswer:` to the impliedAnswerFormat's `stringForAnswer:`.